play-services-core/src/main/res/xml/preferences_start.xml

```diff
@@ -65,9 +65,15 @@
             android:title="@string/service_name_location"/>
     </PreferenceCategory>
     <PreferenceCategory android:layout="@layout/preference_category_no_label" android:key="prefcat_footer">
+        <SwitchPreferenceCompat
+            android:defaultValue="false"
+            android:icon="@drawable/ic_hide_app_icon"
+            android:key="hideAppIcon"
```

Comment:
This doesn't follow our naming convention for preference keys (`pref_<snake_case_name>`).

Reply:
I followed a similar name scheme but changed it in https://github.com/microg/GmsCore/pull/2462/commits/3094aff1c6c6142b612a16ba79936e5849100db2. Reverted that commit.